sdk/storage/azure-storage-common/src/test/java/com/azure/storage/common/StorageSharedKeyCredentialTests.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.common;
+
+import com.azure.core.credential.AzureNamedKeyCredential;
+import com.azure.core.test.TestProxyTestBase;
+import com.azure.storage.common.implementation.StorageImplUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class StorageSharedKeyCredentialTests extends TestProxyTestBase {
```

Comment:
Since none of these tests are running against a service this doesn't need to extend `TestProxyTestBase` and can be a standalone test class. This likely holds true for all tests in `azure-storage-common` as it isn't tied to an Azure Storage service and is shared common functionality.